cmflib/commands/artifact/list.py

```diff
@@ -17,69 +17,217 @@
 import argparse
 import os
 import pandas as pd
+import textwrap
```

Comment:
Do we need to add this library to be part of CMF installation ?

Reply:
Yes, we need to. Good catch.